unit-tests/unit-tests-live.cpp

```diff
@@ -3823,6 +3823,108 @@ TEST_CASE("Per-frame metadata sanity check", "[live][!mayfail]") {
     }
 }
 
+TEST_CASE("color sensor API", "[live][options]")
+{
+    rs2::context ctx;
+    if (make_context(SECTION_FROM_TEST_NAME, &ctx, "2.17.1"))
+    {
+        rs2::device dev;
+        rs2::pipeline pipe(ctx);
+        rs2::config cfg;
+        rs2::pipeline_profile profile;
+        REQUIRE_NOTHROW(profile = cfg.resolve(pipe));
+        REQUIRE(profile);
+        REQUIRE_NOTHROW(dev = profile.get_device());
+        REQUIRE(dev);
+        dev_type PID = get_PID(dev);
```

Comment:
Using pipe can be avoided here as you'd probably iterate over `ctx.query_devices()` list